ci-operator/config/openstack-k8s-operators/horizon-operator/openstack-k8s-operators-horizon-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+build_root:
+  from_repository: true
+images:
+- dockerfile_literal: |
+    FROM src
+    RUN pip3 install pre-commit
+  from: src
+  to: openstack-operator-ci-pre-commit
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make gotest
+  container:
+    from: src
+- as: govet
+  commands: make govet
+  container:
+    from: src
+- as: golangci
```

Comment:
govet and golanci-lint are already included in pre-commit tests, so we can remove both tests from this file[1]

[1] https://github.com/openstack-k8s-operators/horizon-operator/blob/f08ff4a882f55b09395d6ed842c1375044bfe8c7/.pre-commit-config.yaml#L23-L35

Reply:
I have removed from here only